Dockerfile

```
@@ -56,3 +56,10 @@ RUN git clone --branch master https://github.com/pytorch/fairseq.git /fairseq
 WORKDIR /fairseq
 RUN pip install pytest pyyaml
 RUN pip install -e .
+
+FROM base as pytorch-lightning
+RUN git clone --branch 0.9.0 https://github.com/PyTorchLightning/pytorch-lightning /pytorch-lightning
+WORKDIR /pytorch-lightning
+RUN pip install --requirement ./requirements/base.txt --upgrade && pip install cmake
+ENV LC_ALL=C.UTF-8
+RUN sh -c 'apt-get install -y libblas-dev liblapack-dev gfortran && HOROVOD_BUILD_ARCH_FLAGS="-mfma" pip install -e ".[dev]" --global-option="--quiet" --upgrade'
```

```suggestion
RUN sh -c 'apt-get install -y libblas-dev liblapack-dev gfortran && HOROVOD_BUILD_ARCH_FLAGS="-mfma" pip install -e ".[dev]" --upgrade'
```

Let's just get rid of the `--global-option` altogether, it seems to be blocking our usage of wheels and I don't think CircleCI really cares if `pip` is going to be spamming our logs

---

lets see, but for me building locally with removing this option I get following error:
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-mjegbf6u/horovod/setup.py", line 166, in <module>
        'horovodrun = horovod.runner.launch:run_commandline'
      File "/usr/local/lib/python3.6/dist-packages/setuptools/__init__.py", line 153, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/usr/local/lib/python3.6/dist-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/usr/lib/python3.6/distutils/command/install.py", line 589, in run
        self.run_command('build')
      File "/usr/lib/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/usr/lib/python3.6/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/usr/lib/python3.6/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/usr/local/lib/python3.6/dist-packages/setuptools/command/build_ext.py", line 79, in run
        _build_ext.run(self)
      File "/usr/lib/python3.6/distutils/command/build_ext.py", line 339, in run
        self.build_extensions()
      File "/tmp/pip-install-mjegbf6u/horovod/setup.py", line 87, in build_extensions
        cwd=self.build_temp)
      File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release', '--', '-j8', 'VERBOSE=1']' returned non-zero exit status 2.
```

---

so that case was not about talking too much but about Horovod has a bug in interpreting VERBOSE wich shall be fixed in next release... @tgaddair